### PR TITLE
Implemented a basic admin controller class.

### PR DESCRIPTION
**Pull request by Miracle2001**

### CODE REVIEW

**Review by wasimghazal — APPROVED** [review]

**Review by nadinefrawley — CHANGES_REQUESTED** [review]
You need to add in a log in option for admin as they may not be able to register but they should be able to login

File: system/controllers/admin_controller.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import tkinter as tk
+from system import views
+from system.database.db_handler import ProductDB
+from system.controllers.abstract_controllers import AbstractAdminController
+
+
+class AdminController(AbstractAdminController):
+    def __init__(self, root, controller):
```
Comment (dylancoffey99) [inline system/controllers/admin_controller.py:8]: Same changes that I suggested to Wasim, rename "controller" to "access_controller" so there is no confusion.
Also, pass in the user that is created in the access_controller.

So overall it should be:
def __ init __ (self, root, user, access_controller):

File: system/controllers/admin_controller.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        self.root = root
+        self.user = 'Admin'
+        self.controller = controller
```
Comment (dylancoffey99) [inline system/controllers/admin_controller.py:11]: Initialise those changes as so:
self.user = user
self.access_controller = access_controller

**Review by dylancoffey99 — CHANGES_REQUESTED** [review]
AdminView will also need to be updated to update the title of the window to include the logged in users name. See Wasims CustomerView changes to understand.

**Review by dylancoffey99 — APPROVED** [review]
Looks good. We will merge this branch as it's outdated and can't use the OrderDB yet.

**Review by wasimghazal — APPROVED** [review]

**Pull request merged**